extensions/amp-story/1.0/amp-story-reaction.js

```diff
@@ -605,35 +453,32 @@ export class AmpStoryQuiz extends AMP.BaseElement {
    * @private
    */
   executeReactionRequest_(requestOptions, reactionValue) {
-    // TODO(jackbsteinberg): Add a default reactions endpoint.
     if (!assertAbsoluteHttpOrHttpsUrl(this.element.getAttribute('endpoint'))) {
       return Promise.reject(ENDPOINT_INVALID_ERROR);
     }
 
     if (this.reactionId_ === null) {
-      const quizPageId = closest(dev().assertElement(this.element), (el) => {
+      const pageId = closest(dev().assertElement(this.element), (el) => {
         return el.tagName.toLowerCase() === 'amp-story-page';
       }).getAttribute('id');
 
-      this.reactionId_ = `CANONICAL_URL#page=${quizPageId}`;
+      this.reactionId_ = `CANONICAL_URL#page=${pageId}`;
     }
 
     const requestVars = dict({
-      'reactionType': STORY_REACTION_TYPE_QUIZ,
+      'reactionType': this.reactionType_,
       'reactionId': this.reactionId_,
     });
 
     let url = this.element.getAttribute('endpoint');
 
-    if (requestOptions['method'] === 'POST') {
-      requestVars['reactionValue'] = reactionValue;
-      requestOptions['body'] = requestVars;
-    } else if (requestOptions['method'] === 'GET') {
-      url = addParamsToUrl(url, requestVars);
-    }
-
     return this.getClientId_().then((clientId) => {
       requestVars['clientId'] = clientId;
+      if (requestOptions['method'] === 'POST') {
```

Comment:
Will update http calls in future PR according to design doc. This change makes the GET and POST with same params for consistency, and adds the body to the POST request as well